pandas/tests/window/test_rolling.py

```diff
@@ -1189,7 +1188,7 @@ def test_rolling_sem(frame_or_series):
 
 
 @pytest.mark.xfail(
-    (is_platform_arm() and not is_platform_mac()) or is_platform_power(),
```

Comment:
I guess we should remove the `not`. It might be a typo from https://github.com/pandas-dev/pandas/pull/41982. I haven't got the time to look into it again. But I assume this test would fail on all arm chip Mac.

Reply:
I agree. That is what I have it set to with this PR. The new condition is:

```python
is_platform_arm() or is_platform_power()
```

I think the condition `is_platform_arm()` should catch all arm chip Mac. Let me know if I have this wrong.